### PR TITLE
feat: add gifs for kpm oci registry.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):

docs

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/kpm_oci-zh.md [hunk 1]
```diff
@@ -2,75 +2,110 @@
...
+<img src="./gifs/kpm_login_with_both.gif" width="600" align="center" />
 
 ### kpm logout
 
-使用 kpm 登出 OCI registry
+你可以使用 `kpm logout` 退出一个 OCI Registry。
 
 ```shell
-kpm logout <registry>
+$ kpm logout <registry>
```
Comment (Peefy) [inline docs/kpm_oci-zh.md:65]: For one-line shell scripts, do not need to add `$`

File: docs/kpm_oci-zh.md [hunk 2]
```diff
@@ -2,75 +2,110 @@
...
-- `<kcl_package_version>` kcl 包的版本号，例如：v0.0.1。
+<img src="./gifs/kpm_pull.gif" width="600" align="center" />
 
 ### kpm run
 
 kpm 可以直接通过 OCI 的 url 编译 kcl 包。
 
 ```shell
-kpm run --tag <kcl_package_version> oci://<oci_registry>/<account_name>/<repo_name>
+$ kpm run --tag <kcl_package_version> <oci_url>
```
Comment (Peefy) [inline docs/kpm_oci-zh.md:100]: For one-line shell scripts, do not need to add `$`

File: docs/kpm_oci-zh.md [hunk 3]
```diff
@@ -2,75 +2,110 @@
...
-- `<repo_name>` 是用来保存 kcl 包的仓库名称。
-- `<kcl_package_version>` kcl 包的版本号，例如：v0.0.1。
+<img src="./gifs/kpm_run_oci_url.gif" width="600" align="center" />
 
-如果你的 kcl 是保存在 docker.io 上的，那么 kpm 还支持通过如下方式直接编译 kcl 包。
+另外，你也可以通过 `kpm run` 直接使用本地的 kcl 包。
 
 ```shell
-kpm run <docker.io_account_name>/<repo_name>:<kcl_package_version>
+$ kpm run <oci_ref>
```
Comment (Peefy) [inline docs/kpm_oci-zh.md:108]: For one-line shell scripts, do not need to add `$`

Comment (zong-zhe) [reply docs/kpm_oci-zh.md]: fixed.

File: docs/kpm_oci-zh.md (hunk 2 above)
Comment (zong-zhe) [reply docs/kpm_oci-zh.md]: fixed.

File: docs/kpm_oci-zh.md (hunk 1 above)
Comment (zong-zhe) [reply docs/kpm_oci-zh.md]: fixed.

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**